tests/integ/test_local_mode.py

```diff
@@ -366,3 +366,32 @@ def test_mxnet_local_data_local_script():
         mx.delete_endpoint()
         time.sleep(5)
         fcntl.lockf(local_mode_lock, fcntl.LOCK_UN)
+
+
+@pytest.mark.continuous_testing
+def test_local_transform_mxnet(sagemaker_local_session, tmpdir):
+    data_path = os.path.join(DATA_DIR, 'mxnet_mnist')
+    script_path = os.path.join(data_path, 'mnist.py')
```

Comment:
a lot of these tests seem to do the same boilerplate MXNet/MNIST setup - perhaps something can be refactored?